CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to `ord-schema`
+
+Thanks for helping out! What would you like to do?
+
+* [Report an issue](#report-an-issue)
+* [Suggest an improvement](#suggest-an-improvement)
+* [Make a change](#make-a-change)
+
+## Report an issue
+
+Sorry to hear that you're having trouble! Please [file a bug report](https://github.com/Open-Reaction-Database/ord-schema/issues/new?assignees=&labels=bug&template=bug_report.md&title=) to tell us about the problem.
+
+## Suggest an improvement
+
+Great idea! Please [create a feature request](https://github.com/Open-Reaction-Database/ord-schema/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=) to let us know what you'd like to see.
+
+## Make a change
+
+Excellent! There are a few steps you'll need to follow to get ready to submit changes for review:
+
+1. Make sure your changes are consistent with the [goals](https://ord-schema.readthedocs.io/en/latest/overview.html#goals)
+   and [non-goals](https://ord-schema.readthedocs.io/en/latest/overview.html#non-goals) of the database. If your changes
+   address a long-term need we may ask that it be deferred until sometime in the future. In particular, please avoid
+   changes that significantly increase the complexity of creating or using the data.
+1. Make your changes in a [fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-forks)
+   of the `ord-schema` repository. You can work on GitHub or on your local machine:
+   
+   ### GitHub
+   
+   1. Create a fork of `ord-schema` by clicking the "Fork" button at the top-right of this page. 
+      Note that you only need to do this once---all of your changes can use the same fork.
+   1. Follow the [GitHub flow](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/github-flow)
+      to make changes and send a pull request to the official repository.
+   
+   ### Local machine
+   
+   > :warning: Working on your local machine requires familiarity with
+   > [Git](https://git-scm.com/) or [GitHub Desktop](https://desktop.github.com/).
+
+   1. Create a fork of `ord-schema` and clone it to your local machine 
+      ([instructions](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)).
+   1. Create a new branch and make your changes.
+   1. Commit your changes, push them to your fork on GitHub, and send a pull request to the
```

Comment:
Add step along the lines of:

```
1. Test your changes by rebuilding the package (`python setup.py install`) and running the test script provided in the repo (`./run_tests.sh`). This will be performed automatically when you send a pull request to the repository, but you can save time by making sure the tests will pass beforehand.
```

And is it worth mentioning `pylint` too?

Reply:
Yes, good point. I'll add some steps here and mention the CI tests on the GitHub section.